### DISCUSSION TITLE
Thumbnail creation

### DISCUSSION THREAD

**Discussion by EliShklovsky** [pimcore/pimcore#7920] [Q&A / Support]
Hi, 
Since Pimcore 6.7.1 the Imagick thumb creation for pdf, jpg get:
```
pimcore.ERROR: Unable to load image
pimcore.ERROR: ImagickException: not authorized
```
I found that if you change the policy.xml of Imagemagick to 
`<policy domain="coder" rights="none" pattern="PDF" /> `
works for pdf
But what about other file types?
I found this behavior for Pimcore 6.8.6 and even it happens on 6.8.x-dev https://demo.pimcore.fun/admin - where the pdf does not get the thumbnail.
Did someone encountered this? any Pimcore solution beside changing the policy.xml?

**Comment by AdnanMustafa12** [comment]
Hi @EliShklovsky,
I need your help in configuring pimcore 6.7.1 to an existing project. Can you please help me out or let me know how I can install composer and pimcore?
